src/looprestoration.rs

```diff
@@ -1,13 +1,17 @@
 use crate::include::common::bitdepth::AsPrimitive;
 use crate::include::common::bitdepth::BitDepth;
 use crate::include::common::intops::iclip;
+use crate::include::common::intops::imax;
+use crate::include::common::intops::umin;
 use crate::include::stddef::ptrdiff_t;
 use crate::include::stdint::int16_t;
 use crate::include::stdint::int32_t;
 use crate::include::stdint::uint16_t;
 use crate::include::stdint::uint32_t;
 use crate::src::align::Align16;
 
+use super::tables::dav1d_sgr_x_by_x;
```

Comment:
Can you make it a `crate::` import and put it in the block with the rest?  I think it helps with consistency when there are so many imports generally.

Reply:
Done!